docs/developer/app-store/plugins/oidc.mdx

```diff
@@ -22,7 +22,7 @@ Go to Configuration -> Plugins -> OpenID Connect and fill in the fields:
   keys used to verify any JSON Web Token (JWT) issued by the authorization server (Required for both integration flows).
 - **User info URL:** The URL which can be used to fetch user details by using an access token (Required for flow: _Saleor as a resource server_).
 - **OAuth logout URL:** The logout URL which Saleor will return for frontend's logout request (Optional for flow: _Saleor as a client for authorization server_).
-- **Audience:** The Oauth resource identifier. If provided, Saleor will define the audience for each authorization request. Used to fetch user permissions from OAuth provider and map it to Saleor's permission (Optional for both integration flows).
+- **Audience:** The Oauth resource identifier. If provided, Saleor will define the audience for each authorization request. Used to fetch user permissions from OAuth provider and map it to Saleor's permission (Required for flow: _Saleor as a resource server_).
```

Comment:
Why required? Did we change something in the implementation? 
AFAIR it is required when you want to get permissions from the Oauth provider.  👇 **Use OAuth scope permissions:**